test/cljs/rems/administration/test_create_catalogue_item.cljs

```diff
@@ -46,3 +46,105 @@
+                                                     :title "Performance License"}
+                                                :fi {:attachment-id nil
+                                                     :textcontent "Be fast."
+                                                     :title "Performance License FI"}
+                                                :sv {:attachment-id nil
+                                                     :textcontent "Be fast."
+                                                     :title "Performance License SV"}}}
+                               {:archived false
+                                :enabled true
+                                :id 8
+                                :licensetype "text"
+                                :localizations {:en {:attachment-id nil
+                                                     :textcontent "Be fast."
+                                                     :title "Second License"}
+                                                :fi {:attachment-id nil
+                                                     :textcontent "Be fast."
+                                                     :title "Second License FI"}
+                                                :sv {:attachment-id nil
+                                                     :textcontent "Be fast."
+                                                     :title "Second License SV"}}}]
+                    :organization {:organization/id "nbn"
+                                   :organization/name {:en "NBN"
+                                                       :fi "NBN"
+                                                       :sv "NBN"}
+                                   :organization/short-name {:en "NBN en"
+                                                             :fi "NBN fi"
+                                                             :sv "NBN sv"}}}
+                   {:resid "y"
+                    :licenses [{:archived false
+                                :enabled true
+                                :id 8
+                                :licensetype "text"
+                                :localizations {:en {:attachment-id nil
+                                                     :textcontent "Be fast."
+                                                     :title "Performance License"}
+                                                :fi {:attachment-id nil
+                                                     :textcontent "Be fast."
+                                                     :title "Performance License FI"}
+                                                :sv {:attachment-id nil
+                                                     :textcontent "Be fast."
+                                                     :title "Performance License SV"}}}]}
+                   {:resid "u"}
+                   {:resid "z"}]
+        counts (frequencies (map :resid resources))]
```

Comment:
And here, you would not have to do this if you sent the single count. This duplicates "complex logic".

Reply:
I do not understand this comment, I am sorry

Reply:
Just that if you did the previous, and if you could pass the count simply to `resource-label` as a number, you would not need to duplicate this code in the test, that you must do in the real component.